### PR TITLE
Added Django Debug Toolbar in Debug mode

### PR DESCRIPTION
**Pull request by giocalitri**
#### What are the relevant tickets?

N/A
#### What's this PR do?

Adds Django Debug Toolbar
#### How should this be manually tested?

rebuild the web container then verify that the toolbar works

### CODE REVIEW

**Comment by noisecapella** [comment]
When I open the tab, go to SQL and click 'sel' for example I get an error, just a 403 Forbidden. In the logs I see 

    web_1      | [2016-10-25 18:27:43] WARNING 20 [django.request] csrf.py:103 - [d8846d3d8903] - Forbidden (CSRF token missing or incorrect.): /__debug__/sql_explain/

**Comment by giocalitri** [comment]
I am struggling to understand why it is not working:

screen shot 2016-10-25 at 3 35 23 pm

any suggestion?

**Comment by giocalitri** [comment]
actually it seems that that token is incorrect

**Comment by noisecapella** [comment]
Could just be a bug on their end. Anyway I'm fine merging if you want, it's not essential functionality and everything else appears to work

**Comment by giocalitri** [comment]
I actually do not like it because in edX on master it works just fine...
I would like to make more investigations

**Comment by giocalitri** [comment]
@noisecapella you were right: the problem was with the URL import sequence.
Now it is fixed

File: micromasters/settings.py [hunk 1]
```diff
@@ -498,3 +498,26 @@ def get_var(name, default):
...
+    INSTALLED_APPS += ('debug_toolbar', )
+    # it needs to be enabled before other middlewares
+    MIDDLEWARE_CLASSES = (
+        'debug_toolbar.middleware.DebugToolbarMiddleware',
+    ) + MIDDLEWARE_CLASSES
+
+    def show_toolbar(request):
+        """
+        Custom function needed because of bug in wagtail.
+        Theoretically this bug has been fixed in django 1.10 and wagtail 1.6.3
```
Comment (giocalitri) [inline micromasters/settings.py]: I think we need to check if this is still an issue after we upgrade to django 1.10

**Pull request merged**